docs/source/filters.rst

```diff
@@ -165,6 +169,15 @@ List Operators
            value: ID-321                  ─▶ Values to be compared against
            value_type: swap               ─▶ Switches list comparison order
 
+  The above filter can also be written as:
```

Comment:
maybe we should just remove the above example, it's much less intuitive after you realize `contains` exists

Reply:
or swap the two so that the contains example is first

Reply:
Wasn't sure of a way to clearly explain `op: difference` / `op: contains` / `value_type: swap` without pinning it to an example, so clustered example filters around a set of fake resources. Rendered preview from a local build:

![image](https://github.com/cloud-custodian/cloud-custodian/assets/19539955/28163749-6d5c-4b0d-bc2c-8734940ce236)